frontend/integration-tests-cypress/cypress/support/cy-command-additions/index.ts

```diff
@@ -0,0 +1,18 @@
+import './project';
+import './selectors';
+import './nav';
+
```

Comment:
Thoughts on:

```typescript
// selectors.ts
declare global {
  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
  namespace Cypress {
    interface Chainable<Subject> {
      getElemByDataId(selector: string, timeout?: number): Chainable<Element>;
      getElemByActionId(selector: string): Chainable<Element>;
    }
  }
}

// project.ts
declare global {
  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
  namespace Cypress {
    interface Chainable<Subject> {
      createTestProject(testName: string): Chainable<Element>;
      deleteTestProject(testName: string): Chainable<Element>;
    }
  }
}

// navigation.ts
declare global {
  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
  namespace Cypress {
    interface Chainable<Subject> {
      clickNavLink(path: [string, string]): Chainable<Element>;
    }
  }
}
```
Just another question of growth over time.

Reply:
I believe typescript allows this, and will combine them for you.

Reply:
fixed, thanks, good idea, keep interface with methods together